src/orders/OrderProcessor.sol

```diff
@@ -153,6 +158,8 @@ abstract contract OrderProcessor is AccessControlDefaultAdminRules, Multicall, S
     /// @inheritdoc IOrderProcessor
     mapping(address => uint256) public maxOrderDecimals;
 
+    mapping(address => bool) public userHasFeeSchedule;
```

Comment:
Rather than a single feeSchedule for all accounts, a feeSchedule mapping should be address => address, where address(0) replaces userHasFeeSchedule and allows for multiple custom fee schedules